airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -82,6 +83,18 @@ class DAGDetailSchema(DAGSchema):
     doc_md = fields.String(dump_only=True)
     default_view = fields.String(dump_only=True)
     params = fields.Dict(dump_only=True)
+    is_paused = fields.Method("get_is_paused", dump_only=True)
+    is_active = fields.Method("get_is_active", dump_only=True)
+
+    @staticmethod
+    def get_is_paused(obj: DAG):
```

Comment:
an open question: do we need two methods here? It's a "yes or no", so maybe just `get_is_active()` + `not get_is_active()` would already suffice?

Reply:
I think that being `not active` is different from being `paused`, so we will need both methods here. (I could be misinterpreting your question here though)

Reply:
Yeah, an inactive dag is one where the file no longer exists essentially.

Reply:
yep I asked a silly question. please ignore this.